src/test/java/io/fusionauth/samlv2/service/DefaultSAMLv2ServiceTest.java

```diff
@@ -105,6 +110,23 @@
 @SuppressWarnings({"unchecked"})
 @Test(groups = "unit")
 public class DefaultSAMLv2ServiceTest {
+  @DataProvider(name = "assertionEncryption")
+  public Object[][] assertionEncryption() {
```

Comment:
It's not feasible to cover all the possible combinations, but I tried to get at least a couple different values from each enum to make sure they work together.